datasource/tests/tests.py

```diff
@@ -302,3 +306,43 @@ def test_recommend_brands(self):
         self.assertTrue(
             SuggestedAssociation.objects.get(brand=brand2, datasource=usnic5).certainty > 0
         )
+
+
+class FixtureLoadingTestCase(TestCase):
+    def setUp(self):
+        """
+        This method will generate the sample_initial.json file which contains exactly one entry
+        for each module. The size of this file will be very small as compared to the original
+        initial.json
+        """
+
+        file = open("fixtures/initial/initial.json", "r")
+        json_file = json.load(file)
+
+        sample_initial_json = list({data["model"]: data for data in json_file}.values())
+
+        for data in sample_initial_json:
+            data["pk"] = 1
+            for field, field_val in data["fields"].items():
+                if isinstance(field_val, int):
+                    data["fields"][field] = 1
+
+        with open("fixtures/initial/sample_initial.json", "w") as file:
+            json.dump(sample_initial_json, file)
+
+    def tearDown(self):
+        """
+        Just remove the sample_initial.json file after executing the test case.
+        """
+        os.remove("fixtures/initial/sample_initial.json")
+
+    def test_can_import_fixtures(self):
+        """
+        Verify that it is always possible to import initial.json file
+        """
+        try:
```

Comment:
This works

Reply:
I have incorporated the changes, could you please review it?